addons/base_import_module/models/ir_module.py

```diff
@@ -257,6 +259,10 @@ def _import_module(self, module, path, force=False, with_demo=False):
         for bundle, commands in terp.get('assets', {}).items():
             for command in commands:
                 directive, target, path = IrAsset._process_command(command)
+                if is_wildcard_glob(path):
+                    raise UserError(_(
```

Comment:
I don't get it. It was silent until now, and those assets were not loaded. Why is it necessary to raise now (which prevents installation) ? And if there is a real issue, shouldn't it be backported to stable versions as well?

Reply:
I tried to install a module with wildcards in assets. And after installation, the UI is blocked.


> It was silent until now

I think it mainly becase our industry modules never use those wildcards. And they don't have so many customized static codes.

Reply:
OK thanks for the info. I guess if someone ever complains we can backport then :shrug: